web/src/routes/(user)/albums/+page.ts

```diff
@@ -1,17 +1,18 @@
+import { albumListingStore } from '$lib/stores/album-listing.store';
 import { authenticate } from '$lib/utils/auth';
 import { getFormatter } from '$lib/utils/i18n';
-import { getAllAlbums } from '@immich/sdk';
 import type { PageLoad } from './$types';
 
 export const load = (async () => {
   await authenticate();
-  const sharedAlbums = await getAllAlbums({ shared: true });
-  const albums = await getAllAlbums({});
+  const { isCached } = await albumListingStore.getAlbums();
   const $t = await getFormatter();
 
+  // The album data might be old, refetch
+  // non-awaited async
+  if (isCached) albumListingStore.refetchAlbums();
```

Comment:
We try to use the convention of new line with curly braces across the codebase
```suggestion
  if (isCached) {
    albumListingStore.refetchAlbums();
  }
```

Reply:
done :)